manuscript.qmd

```diff
@@ -17,137 +17,277 @@ format:
-        department: Department of Computing
-        address:
-        city: Turku
-        postal-code: 20014
-    email: tvborm@utu.fi
+    orcid: 0000-0002-8563-8884
+    email: tuomas.v.borman@utu.fi
+    affiliations: [1]
     attributes:
       corresponding: true
+
   - name: Giulio Benedetti
-    affiliations:
-        id: 1
-        orcid: 0000-0002-8732-7692
+    orcid: 0000-0002-8732-7692
+    email: giulio.benedetti@utu.fi
+    affiliations: [1]
     attributes:
       equal-contributor: true
+
   - name: Geraldson Muluh
-    affiliations:
-        id: 1
+    orcid: 0000-0002-9721-5192
+    email: geraldson.muluh@utu.fi
+    affiliations: [1]
     attributes:
       equal-contributor: true
+
   - name: Aura Raulo
-    affiliations:
-        id: 1
+    orcid: 0000-0003-4860-7840
+    email: aura.raulo@utu.fi
+    affiliations: [3, 4]
+
   - name: Benjamin Valderamma
-    affiliations:
-      - name: Organization XK
-        id: 2
-        department: Department of Y
-        address:
-        city: City
-        postal-code: 0000000
+    orcid: 0000-0002-4740-8965
+    email: 123116261@umail.ucc.ie
+    affiliations: [5, 6]
+
   - name: Artur Sannikov
-    affiliations:
-      - id: 1
+    orcid: 0000-0001-7765-123X
+    email: artur.sannikov@utu.fi
+    affiliations: [2]
+
   - name: Stefanie Peschel
-    affiliations:
-      - name: Organization D
-        id: 4
-        department:
-        address:
-        city:
-        postal-code:
-    email:
+    orcid: 0000-0002-7936-7627
+    email: stefpeschel@gmail.com
+    affiliations: [7]
+
   - name: Yihan Liu
-    affiliations:
-      - name: Cornell
-        id: 5
-        department: Department of Y
-        address: Address
-        city: City
-        postal-code: 0000000
+    orcid: 0009-0008-0462-7501
+    email: yihan.liu@yale.edu
+    affiliations: [8, 9, 10, 11, 12]
+
   - name: Rasmus Hindström
-    affiliations:
-        id: 1
-        orcid: 0009-0004-5731-178X
+    orcid: 0009-0004-5731-178X
+    email: rasmus.hindstrom@utu.fi
+    affiliations: [1]
+
   - name: OMA consortium
-    affiliations:
-      - id: 6
+    affiliations: [19]
+
   - name: Katariina Pärnänen
-    affiliations:
-      - id: 1
-      - name: University of Helsinki
-        id: 6
-        department: Department of Microbiology
-        address:
-        city: Helsinki
-        postal-code: 20014
+    orcid: 0000-0003-2220-1738
+    email: katariina.parnanen@helsinki.fi
+    affiliations: [13]
+
   - name: Christian L. Müller
-    affiliations:
-        - id: 4
-  - name: Aki Havulinna
-    affiliations:
-        - id: 1
+    orcid: 0000-0002-3821-7083
+    email: christian.mueller@helmholtz-munich.de
+    affiliations: [14, 7, 15]
+
+  - name: Aki S. Havulinna
+    orcid: 0000-0002-4787-8959
+    email: aki.havulinna@utu.fi
+    affiliations: [1, 16]
+
   - name: Sudarshan Shetty
+    orcid: 0000-0001-7280-9915
+    email: sudarshanshetty9@gmail.com
+    affiliations: [17]
+
   - name: Marcel Ramos
```

Comment:
Every author must have an affiliation. If someone hasn't marked that then use "No affiliation."